src/test/functions/env.spec.ts

```diff
@@ -0,0 +1,352 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as os from "os";
+import { sync as rimraf } from "rimraf";
+import { expect } from "chai";
+
+import * as env from "../../functions/env";
+
+describe("functions/env", () => {
+  describe("parse", () => {
```

Comment:
You might want to consider a table-based test (e.g. programmatically drive tests from an array of data) just to cut down on repetitive test code, but honestly this is pretty readable and you've already done the heavy lifting so I think it's also fine as-is.

Reply:
Tried it. Feels prretty good.